h/static/scripts/directive/annotation.coffee

```diff
@@ -45,6 +45,7 @@ AnnotationController = [
     @hasDiff = false
     @showDiff = undefined
     @timestamp = null
+    @element_created_at = new Date().getTime()
```

Comment:
Heh. I just used Date because it seemed to be easier to get in JS than a UUID. Happy to replace it